### ISSUE TITLE
Unknown Auth_In

### ISSUE THREAD

**Issue by dejavu17** [cizaquita/FakeSAMP#4]
Hello can you update fakesamp to work for R3 servers. the server i play just updated their server to R3 and it says Unknown Auth_In! when i login.

**Comment by Mr4sR** [comment]
Maybe the server has an anti-bot system

**Comment by cizaquita** [comment]
Yes, I think that we need a keyfor the version that you are using.

**Comment by cizaquita** [comment]
Working on 0.3.7
Greets!
